man/crystal.1

```diff
@@ -0,0 +1,259 @@
+.\"Crystal Programming Language
+.Dd February 11, 2016
+.Dt CRYSTAL(1) "" "Crystal Programmers Reference Guide"
+.\".Dt CRYSTAL 1
+.Os UNIX
+.Sh NAME
+.Nm crystal
+.Nd Compiler for the statically type-checked programming language Crystal.
```

Comment:
Not sure we need to describe much what Crystal is here. If we decide to do so, I'd do it in a distinct, more elaborate paragraph.